hack/build/docker/builder/Dockerfile

```diff
@@ -52,7 +52,19 @@ COPY output-bazel-arch.sh /output-bazel-arch.sh
 
 RUN curl -L -o /usr/bin/bazel https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-$(sh /output-bazel-arch.sh) && chmod u+x /usr/bin/bazel
 
-# Until we use a ver including the fix for this Bazel issue:
+
+RUN if test "$(sh /output-bazel-arch.sh)" = x86_64; then \
```

Comment:
@cfilleke am I missing something or this should not be x86_64?

Reply:
When the builder container has architecture x86_64, then we add cross-compile libraries, so that when the builder container is running on x86, it builds, via cross-compilation using the libraries included,  CDI operator containers with the s390x exec architecture.  That would be case 'C' in the attached diagram. 

<img width="394" alt="Screenshot 2024-08-21 at 4 04 07 PM" src="https://github.com/user-attachments/assets/afc938c7-3814-45ee-b0c6-d4f17c4bac4e">


Theoretically, you could have all the CDI builder containers enabled to do cross-compilation to produce CDI operators of any other architecture; however, I think the logic behind having only the x86_64 builder enabled to do cross-compilation is to limit this to the cases we actually need.

Reply:
gotcha - I'm happy if you just include a short comment about this above the condition